graylog2-web-interface/src/pages/ShowContentPackPage.jsx

```diff
@@ -35,7 +36,10 @@ const ShowContentPackPage = createReactClass({
 
 
   componentDidMount() {
-    ContentPacksActions.get(this.props.params.contentPackId);
+    ContentPacksActions.get(this.props.params.contentPackId).catch(() => {
+      UserNotification.error('Cannot find a matching Content Pack.');
```

Comment:
The error message is somehow confusing for me, I think because of the "matching" in there. As far as I could think, most times the user will find a 404 it is because the content pack was deleted, so maybe we can change that message into something like:

```
UserNotification.error(`The Content Pack with ID "${id}" was not found and may have been deleted.`, 'Cannot find the Content Pack')
```